network-api/networkapi/tests.py

```diff
@@ -109,8 +110,14 @@ def test_mozilla_foundation_users_not_deleted(self):
 
 
 class RedirectDefaultSiteDecoratorTests(TestCase):
+
     def setUp(self):
         self.factory = RequestFactory()
+        # Change the default site away from localhost
+        self.original_default_site = Site.objects.get(is_default_site=True, hostname='localhost')
+        self.original_default_site.is_default_site = False
```

Comment:
I added this to "remove" the default site. Originally, `localhost` is the default site. These few lines unset `localhost` as the default site so we can add a custom one.